multi-bracket-validation/multi-bracket-validation.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import Stack from './lib/stack';
+
+const multiBracketValidation = (str) => {
+  const vStack = new Stack();
+  const len = str.length;
+  for (let i = 0; i < len; i++) {
+    if (str[i] === '(' || str[i] === '[' || str[i] === '{') {
+      vStack.push(str[i]);
+    }
+    if (str[i] === ')' || str[i] === ']' || str[i] === '}') {
+      const leftBracket = vStack.pop();
+      if (leftBracket === '(' && str[i] !== ')') {
+        return false;
+      }
+      if (leftBracket === '[' && str[i] !== ']') {
+        return false;
+      }
+      if (leftBracket === '{' && str[i] !== '}') {
+        return false;
+      }
+    }
+  }
+
+  if (vStack.storage.length > 0) {
```

Comment:
This if statement is totally fine. Another shorthand way when returning true/false with this condition would be something like:
```
return vStack.storage.length > 0
```

The above will return true or false based off the `vStack.storage.length` condition.